indico/core/settings/proxy.py

```diff
@@ -203,3 +203,19 @@ def __set__(self, obj, value):
 
     def __delete__(self, obj):
         self.proxy.delete(*self._make_args(obj, self.name))
+
+
+class ProxyProperty(object):
```

Comment:
> ProxyProperty - proxy property that maps to an attribute of a proxied attribute;

Isn't it more like "proxy property that maps to an attribute of a proxied **class**?"

In any case, following the `SettingProperty` naming it's true that `AttributeProperty` keeps consistency. OTOH, it's very hard to imagine what a class named `AttributeProperty` does, at least for me. I'd suggest `AttributeProxyProperty` (which would also require the renaming of `SettingProperty` to `SettingProxyProperty`).